src/utils/toast.ts

```diff
@@ -0,0 +1,5 @@
+import { toast } from "react-toastify";
```

Comment:
You can leave this for now, but it's a good practice to avoid adding too many dependencies to a project. You're already using mui/joy, which has its own Toast component, so adding react-toastify isn't really necessary